sljit_src/sljitWXExecAllocator.c

```diff
@@ -64,17 +64,12 @@ SLJIT_API_FUNC_ATTRIBUTE void sljit_free_exec(void* ptr)
 
 static void sljit_update_wx_flags(void *from, void *to, sljit_s32 enable_exec)
 {
-	static sljit_uw page_mask = 0;
+	sljit_uw page_mask = (sljit_uw)get_page_alignment();
 
 	sljit_uw start = (sljit_uw)from;
 	sljit_uw end = (sljit_uw)to;
 	int prot = PROT_READ | (enable_exec ? PROT_EXEC : PROT_WRITE);
 
-	if (page_mask == 0) {
-		/* This is thread safe, since the value is constant. */
-		page_mask = (sljit_uw)(sysconf(_SC_PAGESIZE) - 1);
```

Comment:
include is no longer needed after

https://github.com/zherczeg/sljit/blob/cdd8a685014af2680ac27a0ffe315b89cdc9d4c6/sljit_src/sljitWXExecAllocator.c#L34